doc/Language/iterating.pod6

```diff
@@ -4,136 +4,321 @@
 
 =SUBTITLE Functionalities available for visiting all items in a complex data structure
 
-=head1 The C<Iterator> and C<Iterable> roles
+Similar to other many programming languages, Raku makes a fine distinction among
```

Comment:
There's no good reason to eliminate a header. You can change its name, but it helps index the page and navigation.